Raven.Tests.Helpers/RavenTestBase.cs

```diff
@@ -138,6 +138,24 @@ protected string NewDataPath(string prefix = null, bool forceCreateDir = false)
 			return newDataDir;
 		}
 
+        /// <summary>
+        /// Creates a new Embeddable document store.
+        /// </summary>
+        /// <param name="runInMemory">Whatever the database should run purely in memory. When running in memory, nothing is written to disk and if the server is restarted all data will be lost.<br/>Default: <b>true</b></param>
+        /// <param name="requestedStorage">What storage type to use (see: RavenDB Storage engines).<br/>Allowed values: <b>esent</b>, <b>munin</b>.<br/>Default: <b>esent</b></param>
+        /// <param name="catalog"></param>
+        /// <param name="dataDir">The path for the database directory. Can use ~\ as the root, in which case the path will start from the server base directory. <br/>Default: <b>~\Data</b></param>
+        /// <param name="enableAuthentication"></param>
+        /// <param name="activeBundles"></param>
+        /// <param name="port">The port to use when creating the http listener. Allowed: 1 - 65,536 or * (find first available port from 8079 and upward).<br/>Default: <b>8079</b></param>
+        /// <param name="anonymousUserAccessMode">Determines what actions an anonymous user can do. Get - read only, All - read & write, None - allows access to only authenticated users, Admin - all (including administrative actions).<br/>Default: <b>Get</b></param>
+        /// <param name="configureStore"></param>
```

Comment:
configureStore is an Action delegate, that allow you to configure the DocumentStore instance that you get back.